scripts/missingRules.ts

```diff
@@ -0,0 +1,60 @@
+/* tslint:disable: no-console */
+import 'isomorphic-fetch';
+import {JSDOM} from 'jsdom';
+import {red, green} from 'chalk';
+import {emoji} from 'node-emoji';
+
+import functionalityRules from '../src/rules/functionality';
+import maintainabilityRules from '../src/rules/maintainability';
+import styleRules from '../src/rules/style';
+import typescriptRules from '../src/rules/typescript';
+
+const currentRulesArr = Object.keys({
+  ...functionalityRules,
+  ...maintainabilityRules,
+  ...styleRules,
+  ...typescriptRules,
+});
+
+const tslintRulesURL = 'https://palantir.github.io/tslint/rules/';
+fetch(tslintRulesURL, {'method': 'GET'})
+  .then((res) => res.text())
+  .then(printMissingRules);
+
+function printMissingRules(tslintRulesHTML) {
+  const missingRules = getTslintRules(tslintRulesHTML).filter(isRuleMissing);
+  if (missingRules.length === 0) {
+    console.log(green.bold(`${emoji.smile} Rules are all up to date!`));
+    return;
+  }
+
+  console.log(red.bold(`\n${emoji.cry}  MISSING RULES:`));
+  missingRules.forEach((rule) => {
+    console.log(`\n${green.bold('Rule:')} ${rule.name}`);
+    console.log(`${green.bold('Doc:')} ${rule.description}`);
+  });
+}
+
+function isRuleMissing(tslintRule) {
+  return currentRulesArr.indexOf(tslintRule.name) === -1;
```

Comment:
@GoodForOneFare `ts-node` did automatically read main `tsconfig.sjon`. However, VSCode wasn't picking up the `missingRules.ts` script since it wasn't in the `include` list. This resulted in the screenshot above.